assets/dev/scss/frontend/_container.scss

```diff
@@ -175,10 +175,11 @@
 			flex-basis: 100%;
 		}
 
-		// Fix media carousel widget broken (ED-6796).
-		// TODO: Find a better solution.
-		&-media-carousel {
+		// Fix swiper-based widgets broken (ED-6849).
+		// TODO: Use `:has( .swiper-container )` selector when available && remove this class from `frontend/utils/swiper.js`.
```

Comment:
Why not use the existing `e-swiper-container` class

Reply:
Because it's attached to the column